bin/github-backup

```diff
@@ -537,6 +537,24 @@ def _request_url_error(template, retry_timeout):
     return False
 
 
+def download_file(url, path, auth):
+    request = Request(url)
+    request.add_header('Accept', 'application/octet-stream')
+    request.add_header('Authorization', 'Basic '.encode('ascii') + auth)
+    data = urlopen(request)
+    with open(path, 'wb') as f:
+        f.write(data.read())
+
+    # import requests
```

Comment:
This works, but right now `python-github-backup` is a pure python project. I just wanted to illustrate my theory that `urllib` is at fault.